c2corg_ui/lib/cacheversion.py

```diff
@@ -0,0 +1,41 @@
+# Inspired from c2cgeoportal
+# https://github.com/camptocamp/c2cgeoportal/blob/master/c2cgeoportal/lib/cacheversion.py
+
+import uuid
+from urllib.parse import urljoin
+
+
+CACHE_PATH = []
+CACHE_VERSION = ''
+
+
+# TODO cache
+def get_cache_version():
+    """Return a cache version that is regenerate after each cache invalidation"""
+    # return CACHE_VERSION
+    return uuid.uuid4().hex
+
+
+def version_cache_buster(request, subpath, kw):  # pragma: nocover
+    return urljoin(get_cache_version() + "/", subpath), kw
```

Comment:
Here you can get the Git revision with:

    request.registry.settings.get('cache_version')